plugins/GSdx/res/tfx.fx

```diff
@@ -92,6 +92,7 @@ cbuffer cb0
 	float4 VertexScale;
 	float4 VertexOffset;
 	float2 TextureScale;
+	float4 TextureOffset;
```

Comment:
Won't be possible to pack `TextureOffset` and `TextureScale` together. The GPU unit is `float4`. Beside it will allow to avoid a GSVector4 check when you udpate the buffer cache.